instances/latest/parcellationEntityVersions/PW-RBSC-cor_6th-ed/PW-RBSC-cor_6th-ed_temporalAssociationCortex.jsonld

```diff
@@ -2,16 +2,16 @@
   "@context": {
     "@vocab": "https://openminds.ebrains.eu/vocab/"
   },
-  "@id": "https://openminds.ebrains.eu/instances/parcellationEntityVersion/PW-RBSC-cor_6th-ed_temporalAssociatinCortex",
+  "@id": "https://openminds.ebrains.eu/instances/parcellationEntityVersion/PW-RBSC-cor_6th-ed_temporalAssociationCortex",
   "@type": "https://openminds.ebrains.eu/sands/ParcellationEntityVersion",
   "abbreviation": "TeA",
   "additionalRemarks": "Brain region name and abbreviation listed in both chapter 'List of Abbreviations' and chapter 'List of Structures' of the 6th edition of The Rat Brain in Stereotaxic Coordinates (ISBN: 0-12-547612-4). Brain region name and abbreviation were taken from chapter 'List of Abbreviations'.",
   "alternateName": null,
   "correctedName": null,
   "hasAnnotation": null,
   "hasParent": null,
-  "lookupLabel": "PW-RBSC-cor_6th-ed_temporalAssociatinCortex",
-  "name": "temporal associatin cortex",
+  "lookupLabel": "PW-RBSC-cor_6th-ed_temporalAssociationCortex",
+  "name": "temporal association cortex",
```

Comment:
This is not correct. For all rat atlases, we followed the logic that the name is the name including typos and other "weird" things e.g., [frontal association cortex](https://github.com/openMetadataInitiative/openMINDS_instances/blob/main/instances/latest/parcellationEntityVersions/PW-RBSC-cor_6th-ed/PW-RBSC-cor_6th-ed_frontalAssociationCortex.jsonld).
Instead, we make use of "correctedName" to state the name that the area should actually receive and comment on the change under "additionalRemarks". 

I actually made the PR for this #137 but forgot to fix BA and BAV. So, we stick to your PR but you can copy the content of the PEV over.